README.md

```diff
@@ -17,6 +17,32 @@ between two git revisions.
 
 It will configure `@owner` as the default owner in the config file.
 
+
+### Generating and validating owners
+
+By default, owners will be validated either against prepopulated file (teams.txt) or against data fetched from github.
+Format of teams.txt is one owner per line.
+
+Example teams.txt:
+```
+@company/backend-devs
+@company/frontend-devs
+@john.smith
+```
+Github credentials are taken from the following environment variables. You might want to put them into your .bashrc (or equivalent):
+
+    $ export GITHUB_TOKEN='your github PAT' # your personal access token from github
+    $ export GITHUB_organization='company' # name of your github organization
```

Comment:
Could we parse it from `git config --get remote.origin.url`? We could make the variable optional.

Also maybe `codeowners-checker config organization` like we do with `owner`?

BTW it's a good habit to use all upper case for global shell variables.